pkg/volume/iscsi/iscsi.go

```diff
@@ -179,6 +179,10 @@ func (b *iscsiDiskBuilder) IsReadOnly() bool {
 	return b.readOnly
 }
 
+func (b *iscsiDiskBuilder) RequiresSELinuxRelabel() bool {
+	return false
```

Comment:
True, theoretically, but I'm not sure we need to test it.  @childsb, @erinboyd, @wattsteve ?